### PR TITLE
Add X500 Build Log

### PR DESCRIPTION
**Pull request by Junkim3DR**
Full build log of X500 frame with Pixhawk4 V5 kit.

### CODE REVIEW

File: Add CUAV V5+ Build Log [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+Key information
+
+- **Frame:** DJI F450
+- **Flight controller:** [CUAV V5+]
+- **Assembly time (approx.):** 1.5 hours (1 for frame, 0.5 autopilot installation/configuration)
+
+## Bill of materials
+
+* Power module holybro
```
Comment (hamishwillee) [inline Add CUAV V5+ Build Log]: Which one? http://docs.px4.io/master/en/power_module/cuav_hv_pm.html ?

File: Add CUAV V5+ Build Log [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+Step 3: Solder the Power Module, positive (red) and negative (black).
+See figure 3.
+
+Step 4: Plug in the motors to the ESCs according to its positions. For example on Figure 4.   
+
+Step 5: Attach the motors to the corresponding arm, see example in figures 5 and 6
+
+Step 6: Place the top board as shown on figure 7
+
+Step 7: Paste the damping foam on the CUAV V5+ flight controller.
```
Comment (hamishwillee) [inline Add CUAV V5+ Build Log]: Is it internally damped? If so, should it not be double sided tape?

**Comment by hamishwillee** [comment]
Let me know when you're ready for review. For now my only comment is 
1. Remember people need to know where to get the bits and pieces, so make it clear with info about what comes with autopilot, what comes in the frame kit etc. Also, for things like escs or motors that you get separately, make it clear what version they are so that people can source the same. If possible make it clear what the *selection criteria were* - e.g. if someone can't get those motors, what alternatives might work.
1.  check out http://docs.px4.io/master/en/frames_multicopter/dji_f450_cuav_5plus.html for an approach to copy.

File: Add CUAV V5+ Build Log (hunk 1 above)
Comment (Junkim3DR) [reply Add CUAV V5+ Build Log]: Power Module used is PM02 for the X500 Buildlog

File: Add CUAV V5+ Build Log (hunk 2 above)
Comment (Junkim3DR) [reply Add CUAV V5+ Build Log]: It is double sided taped.

**Comment by hamishwillee** [comment]
Also see https://github.com/PX4/px4_user_guide/pull/628#issuecomment-567278064

**Comment by Junkim3DR** [comment]
Created a new PR for the new file created at en/frames_multicopter/holybro_s500_v2_pixhawk4_mini.md
#639

Pending to add pictures.

**Comment by hamishwillee** [comment]
@Junkim3DR Yes, that looks much better. Lets close this one.

**Pull request closed without merging**